Mathlib/Topology/UniformSpace/UniformConvergenceTopology.lean

```diff
@@ -871,6 +879,15 @@ theorem t2Space_of_covering [T2Space β] (h : ⋃₀ 𝔖 = univ) : T2Space (α
     exact separated_by_continuous (uniformContinuous_eval_of_mem β 𝔖 hxs hs).continuous hx
 #align uniform_on_fun.t2_space_of_covering UniformOnFun.t2Space_of_covering
 
+/-- The restriction map `(⋃₀ 𝔖).restrict ∘ UniformOnFun.toFun 𝔖` from `α →ᵤ[𝔖] β` to `⋃₀ 𝔖 → β` is
+uniformly continuous. -/
```

Comment:
I agree with you to some extent, although working with beginners at LFTCM made me remember that it takes some work to be able to *read* Lean statements. I'll admit that sometimes I do so just because it takes more energy to write down the full statement without referencing the Lean objects at play.

In this particular case though, I think it looks useless in the source but not so much in the generated documentation because of the lack of type ascriptions. For example you can look at the similar [UniformOnFun.uniformContinuous_restrict](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Topology/UniformSpace/UniformConvergenceTopology.html#UniformOnFun.uniformContinuous_restrict), where the docstring improves readability just by making the types explicit.

Reply:
Although now that I say that, the statement I linked to doesn't write down the Lean term for the restriction map. Should I do the same here?

Reply:
I did that for now, let me know what you think.

Reply:
What you've got is great, I don't think we need to hold up this really useful PR any more by my quibbles.